CMakeLists.txt

```diff
@@ -85,6 +85,9 @@ if(LSTM_TORCH_LIB_ACTIVE)
    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${TORCH_CXX_FLAGS}")
 endif()
 
+find_package(Boost 1.72.0 REQUIRED)
+
+
```

Comment:
Looks like a conflict resolution artifact - this got moved up to to line 76 to be next to the Boost variable handling

Reply:
Shoot, thought I handled them correctly. Thanks for the catch 🤦🏾 haha